jwt.js

```diff
@@ -92,9 +91,25 @@ function fastifyJwt (fastify, options, next) {
     secret: secret,
     sign: sign,
     verify: verify
-  })
-  fastify.decorateRequest('jwtVerify', requestVerify)
-  fastify.decorateReply('jwtSign', replySign)
+  }
+  if (options.namespace) {
+    if (!fastify.jwt) {
+      fastify.decorate('jwt', {})
+    }
+
+    if (fastify.jwt[options.namespace]) {
+      return next(new Error(`JWT namespace already used ${options.namespace}`))
+    }
+    fastify.jwt[options.namespace] = jwtConfig
+  } else {
+    fastify.decorate('jwt', jwtConfig)
+  }
+
+  var jwtVerify = options.jwtVerify || 'jwtVerify'
```

Comment:
```suggestion
  const jwtVerify = options.jwtVerify || 'jwtVerify'
```

If you merge master, the linter should drop the `var` keyword, so could you run `standard --fix`?